scripts/queries/benchmark_queries/benchmark_runner.sh

```
@@ -0,0 +1,112 @@
+# BENCHMARK_BASE_DIR: Base directory for experiment (required)
+# BENCHMARK_FILE: Output CSV file (default: $BENCHMARK_BASE_DIR/benchmark_${BUILD_TYPE}.csv)
+# BENCHMARK_REPEATS: Number of runs per query (default: 3)
+# BENCHMARK_EXECUTABLE: Path to executable (default: build/${BUILD_TYPE}/offline)
+# BENCHMARK_CSV: Data CSV file (required)
+# BENCHMARK_DECLARATION: Declaration file (required)
+# BENCHMARK_EXTRA_ARGS: Extra arguments (e.g., "-o quarantine.core")
+# BENCHMARK_COMPRESSED_CSV: Compressed CSV file to extract (optional)
+# BENCHMARK_CLEANUP_CSV: Clean up CSV after (optional, default: false)
+# BENCHMARK_COMPRESSED_RESULTS: Compressed expected_results to extract (optional)
+
+cd "$(dirname "$0")"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)